### PR TITLE
fix: show notification icon on tablet

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes
- show notification icon in tablet (issue reported here: https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1715498458516419)

Screenshot 2024-05-25 at 7 35 53 AM
Screenshot 2024-05-25 at 7 35 45 AM

### Preview domain
https://fix-notification-tablet.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/feeds/FeedNav.tsx [hunk 1]
```diff
@@ -91,7 +99,7 @@ function FeedNav(): ReactElement {
         </TabContainer>
 
         {isMobile && sortingEnabled && isSortableFeed && (
-          <div className="sticky flex h-11 w-20 -translate-y-12 translate-x-[calc(100vw-100%)] items-center justify-end bg-gradient-to-r from-transparent via-background-default via-40% to-background-default pr-4">
+          <StickyNavIconWrapper className="translate-x-[calc(100vw-100%)]">
```
Comment (rebelchris) [inline packages/shared/src/components/feeds/FeedNav.tsx:102]: Can't we do the className with `translate-x-[calc(100vw-100%)] tablet:translate-x-[calc(100vw-180%)]`

File: packages/shared/src/components/feeds/FeedNav.tsx [hunk 2]
```diff
@@ -108,7 +116,12 @@ function FeedNav(): ReactElement {
               onChange={(_, index) => setSelectedAlgo(index)}
               drawerProps={{ displayCloseButton: true }}
             />
-          </div>
+          </StickyNavIconWrapper>
+        )}
+        {!isMobile && !isLaptop && (
+          <StickyNavIconWrapper className="translate-x-[calc(100vw-180%)]">
+            <NotificationsBell compact />
+          </StickyNavIconWrapper>
```
Comment (rebelchris) [inline packages/shared/src/components/feeds/FeedNav.tsx:124]: This is the only icon we need to add?
Just wondering as there are quite a few additional items right

**Review by rebelchris — APPROVED** [review]
Non blocking comments

File: packages/shared/src/components/feeds/FeedNav.tsx (hunk 1 above)
Comment (nensidosari) [reply packages/shared/src/components/feeds/FeedNav.tsx:102]: It's 2 different icons, this is sorting, the one we're fixing is notifications which show on different places.

Comment (rebelchris) [reply packages/shared/src/components/feeds/FeedNav.tsx:102]: But they both the same component? I mean the StickyNavIconWrapper

Comment (nensidosari) [reply packages/shared/src/components/feeds/FeedNav.tsx:102]: It's the wrapper but it should show on different places for different icons, not on different places for tablet/mobile so this way we can decide where to show the wrapper +component based on the component

File: packages/shared/src/components/feeds/FeedNav.tsx (hunk 2 above)
Comment (nensidosari) [reply packages/shared/src/components/feeds/FeedNav.tsx:124]: Though I can remove the conditional render via js and will hide in laptop and mobile via css

**Pull request merged**